circuits/cpp/barretenberg/cpp/src/barretenberg/honk/composer/standard_composer.test.cpp

```diff
@@ -5,8 +5,8 @@
 
 #include "barretenberg/honk/composer/standard_composer.hpp"
 #include "barretenberg/honk/proof_system/prover.hpp"
-#include "barretenberg/honk/sumcheck/relations/permutation_relation.hpp"
-#include "barretenberg/honk/sumcheck/relations/relation_parameters.hpp"
+#include "barretenberg/proof_system/relations/permutation_relation.hpp"
```

Comment:
I moved the relations to the `proof_system` module because they will eventually be used to implement the `check_circuit` logic, starting with Zac's ECCVM code.